src/test/charts/functest/templates/ingress-EKS.yaml

```diff
@@ -1,7 +1,12 @@
+{{- if eq "EKS" .Values.clusterType }}
```

Comment:
even though the functest Helm chart has both EKS and KITT ingress definitions, only one wil be rendered, based on this conditional